src/ext/java/org/opentripplanner/ext/siri/SiriTimetableSnapshotSource.java

```diff
@@ -1167,6 +1167,16 @@ private void addTripOnServiceDateToBuffer(
         listOfReplacedVehicleJourneys.add(replacedTrip);
       });
 
+      // Add additional replaced service journeys if present.
```

Comment:
ofNullable, "of" will trhow NullPointerException if not present and ifPresent will not even be checked.